apps/dotcom/client/src/tla/utils/app-ui-events.tsx

```diff
@@ -34,6 +39,17 @@ export interface TLAppUiEventMap {
 		padding: TldrawAppUser['exportPadding']
 		background: TldrawAppUser['exportBackground']
 	}
+	'set-shared-link-type': { type: TldrawAppFile['sharedLinkType'] | 'no-access' }
+	'learn-more-sharing-link': null
+	'learn-more-publishing': null
```

Comment:
Might be better to have a `open-url` event with the url as the payload. Something like what we do inside `tldraw`.
https://github.com/tldraw/tldraw/blob/main/packages/tldraw/src/lib/ui/context/events.tsx#L112